src/dso_api/dynamic_api/filters/values.py

```diff
@@ -0,0 +1,136 @@
+"""Conversion of query-string values to Python types."""
+from __future__ import annotations
+
+import math
+import re
+from datetime import date, datetime, time
+from decimal import Decimal
+
+from django.contrib.gis.geos import GEOSException, GEOSGeometry, Point
+from django.utils.dateparse import parse_datetime
+from django.utils.translation import gettext_lazy as _
+from gisserver.geometries import CRS
+from rest_framework.exceptions import ValidationError
+
+# Don't want Decimal("NaN"), Decimal("-inf") or '0.321000e+2' to be accepted.
+RE_DECIMAL = re.compile(r"^[0-9]+(\.[0-9]+)?$")
+
+
+def str2bool(value: str) -> bool:
+    value = str(value).lower()
```

Comment:
This was there to handle unit tests that passed querystring-dicts with boolean values in. Should I remove this, or leave it? because this does fix such corner case.

Reply:
Sounds like the unit test should be fixed instead.